### DISCUSSION TITLE
Fail to compile the codes

### DISCUSSION THREAD

**Discussion by BleachHwang** [Anura3D/Anura3D_OpenSource#36] [Q&A] [answered]

**Comment by BleachHwang** [comment, marked as answer]
I have followed the manual, and installed the VS Community version 2019 and the oneAPI package (Base + HPC Toolkits). However, I failed to compile the codes. "3 succeeded, 1 failed", as can be seen in the screenshot. Could you please tell me the potential causes? Thank you very much.
![image](https://user-images.githubusercontent.com/90396667/194982139-4ab31f14-72c2-4da1-bd27-8af7db81f5b6.png)

**Reply by BleachHwang** [reply]
**This problem is solved. I found a bug in the code. In Module ModEmptyElements (i.e., file MPMEmptyElements), Line 716, a redundant “&” is found. Please delete it then the codes can be compiled.**
![image](https://user-images.githubusercontent.com/90396667/195025903-2d58230c-c035-42fc-9d32-3ab5d3d09156.png)

**Reply by zhangfengeo** [reply]
I met this problem the same with you,fortunately,the code compiled successfully after deleting the redundant “&”.

**Comment by ayerrocolom** [comment]
Hello,
Thank you for your comment! Indeed, "&" in this line is redundant. We have never experienced a compiling issue because of this before. We will remove it from the code. 
Best,
Alba
